templates/main.html

```diff
@@ -0,0 +1,19 @@
+<!-- BEGIN [Main] ----------------------------------------------------------------->
+<main class="container">
+    <article class="column is-10 is-offset-1">
```

Comment:
It's a 12-column system.  Setting the main content to `10` with a `1` offset mean it will be centered with one column of padding on each side.